hack/base-images/README.md

```diff
@@ -0,0 +1,29 @@
+# Create docker image from Centos ISO
+
+This page documents steps required to create minmal docker image from centos 7 ISO
+
+The build scripts are slightly modified version of <https://github.com/CentOS/sig-cloud-instance-build/tree/master/docker>
+Build scripts on the `sig-cloud-instance-build` repository are used to create official centos docker images.
+
+## Prerequisites
+All steps needed to be executed in a Centos VM.
```

Comment:
Do we need to copy the ks file to the new build machine as well?

Reply:
yes. I have added instructions in the readme file to do the same.